src/render_tasks/d3d12_rtao_task.hpp

```diff
@@ -20,11 +20,12 @@ namespace wr
 	{
 		struct Runtime
 		{
-			float bias = 0.01f;
-			float radius = 1.f;
-			float power = 1.f;
-			int sample_count = 8;
-		};
+			float bias = 0.05f;
+			float radius = 2.f;
+			float power = 1.f; // The final AO output is pow(AO, powerExponent) // 1.0~4.0
+			float max_distance = 2000.f;
+			int sample_count = 128;
```

Comment:
128 samples? also didnt you remove the bias?

Reply:
I'll lower it to be courteous to others, bias is not removed